trapi_throttle/utils.py

```diff
@@ -0,0 +1,22 @@
+import asyncio
+
+def all_equal(values: list):
+    """ Check that all values in given list are equal """
+    return all(values[0] == v for v in values)
+
+def get_equal_dict_values(dct: dict):
```

Comment:
The function name does not reflect the docstring and the behavior, for me. I think it would be better to provide that `first_value` as an argument and call this something like `get_keys_with_value(dct: dict, value)`. There would be no reason to return the values, so we could just return a list of keys. I think this would nicely line up with our needs in `process_batch()`.